omc3/optics_measurements/rdt.py

```diff
@@ -123,12 +122,30 @@ def write(df, header, meas_input, plane, rdt):
               save_index="NAME")
 
 
-def _rdt_to_str(rdt):
+@contextmanager
+def _check_amp_error(rdt: RDTTuple):
+    """ Context manager to catch a specific type of ValueError, regarding the AMP_ column not found in the output,
+    as raised by get_line_sign_and_suffix. 
+    
+    Alternatively, we could just raise a custom Error Type? """
+    try:
+        yield 
+    except ValueError as e:
+        error_str = str(e)
+        message  = f"RDT calculation failed for {jklm2str(*rdt)}."
+
+        if "AMP_" not in error_str:  # raise unexpected Value errors 
```

Comment:
Also is this correct? Should we be looking for "AMP" simply?

Reply:
it is correct and I wanted to make it as specific as possible